build.sbt

```diff
@@ -713,36 +713,39 @@ lazy val compiler = configureAsSubproject(project)
   .dependsOn(library, reflect)
 
 lazy val interactive = configureAsSubproject(project)
+  .settings(commonSettings)
   .settings(disableDocs)
   .settings(skip in publish := true)
   .settings(
     name := "scala-compiler-interactive",
-    description := "Scala Interactive Compiler"
+    description := "Scala Interactive Compiler",
+    // TODO remove `-unit-special` once we re-STARR on 2.13.3, because scala/bug#11965
```

Comment:
This is that thing where they call out joke numbers, right?

I keep thinking "Saturday night special", and realized I don't know how I know that term. Wikipedia offers the synonym, "suicide special".